js/exporter/jspdf/pdf_grid.js

```diff
@@ -15,7 +16,7 @@ export class PdfGrid {
 
     startNewTable(drawTableBorder, firstTableRect, firstTableOnNewPage, splitByColumns) {
         this._currentHorizontalTables = [
-            new PdfTable(drawTableBorder, firstTableRect)
+            new PdfTable(drawTableBorder, firstTableRect, this._columnWidths.slice(0))
```

Comment:
PdfTable will receive its width for its columns and for all the rest columns.
Whether it's better to sync the length of the passed array with the number of this table columns?

Reply:
ok